CPT331.Web/Controllers/AdminController.cs

```diff
@@ -191,54 +190,65 @@ public ActionResult Crimes(string sortBy, SortDirection? sortDirection, uint? pa
 					case "IsDeleted":
 						if (sort == SortDirection.Ascending)
 						{
-							crimes = crimes.OrderBy(m => (m.IsDeleted));	//	.ThenBy(m => (m.Name));
+							crimes = crimes.OrderBy(m => (m.IsDeleted));    //	.ThenBy(m => (m.Name));
 						}
 						else
 						{
-							crimes = crimes.OrderByDescending(m => (m.IsDeleted));	//	.ThenBy(m => (m.Name));
+							crimes = crimes.OrderByDescending(m => (m.IsDeleted));  //	.ThenBy(m => (m.Name));
 						}
 						break;
 
 					case "IsVisible":
 						if (sort == SortDirection.Ascending)
 						{
-							crimes = crimes.OrderBy(m => (m.IsVisible));	//	.ThenBy(m => (m.Name));
+							crimes = crimes.OrderBy(m => (m.IsVisible));    //	.ThenBy(m => (m.Name));
 						}
 						else
 						{
-							crimes = crimes.OrderByDescending(m => (m.IsVisible));	//	.ThenBy(m => (m.Name));
+							crimes = crimes.OrderByDescending(m => (m.IsVisible));  //	.ThenBy(m => (m.Name));
 						}
 						break;
 
-					//	case "Name":
-					//		if (sort == SortDirection.Ascending)
-					//		{
-					//			crimes = crimes.OrderBy(m => (m.Name));
-					//		}
-					//		else
-					//		{
-					//			crimes = crimes.OrderByDescending(m => (m.Name));
-					//		}
-					//		break;
+						//	case "Name":
+						//		if (sort == SortDirection.Ascending)
+						//		{
+						//			crimes = crimes.OrderBy(m => (m.Name));
+						//		}
+						//		else
+						//		{
+						//			crimes = crimes.OrderByDescending(m => (m.Name));
+						//		}
+						//		break;
 				}
 			}
 
 			return View(crimes);
 		}
         #endregion
 
-        #region NewLocalGovernmentArea Model
+        #region LocalGovernmentArea Model
         /// <summary>
-        /// Provides an editor page for an existing LocalGovernmentArea record.
+        /// Exports the ExportLocalGovernmentAreas table as CSV.
         /// </summary>
-        /// <param name="id">The unique ID for the LocalGovernmentArea record.</param>
-        /// <returns>The 'Edit LocalGovernmentArea' view.</returns>
-        /// <permission cref="AdminAuthorizeAttribute">Only authorised personnel have access to this method.</permission>
+        /// <returns>LocalGovernmentAreas.csv</returns>
         [HttpGet]
-        public ActionResult LocalGovernmentArea(uint id)
+		public ActionResult ExportLocalGovernmentAreas()
+
+		{
+			return new ExportDataActionResult("LocalGovernmentAreas.csv", DataProvider.LocalGovernmentAreaRepository);
```

Comment:
Perhaps not the most useful information without the area column.

Maybe we should use the area.STText() in the proc after all?